app/components/SearchResultsGrid.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import _ from 'lodash';
+import React from 'react';
+import {Link} from 'react-router';
+import {processBasicInfoRow, getImageUrl} from '../core/utils';
+import InfoRow from './InfoRow';
+
+
+function getInfoRow(row) {
+  delete row.boxContent;
```

Comment:
To remove the infoBox ([ year -> year ]) from the row. I though about leaving the infoBox, but without it the contents in the page breath better, and it's closer to the original design.